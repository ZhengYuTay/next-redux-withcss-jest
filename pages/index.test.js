import { shallow } from "enzyme";
import React from "react";
import renderer from "react-test-renderer";

import App from "./index.js";

describe("With Enzyme", () => {
  it('App shows "O Hai world!"', () => {
    const app = shallow(<App />);
    console.log('here ',app.find(".example").text());
    expect(app.find(".example").text()).toEqual("O Hai world!");
  });
});

describe("With Snapshot Testing", () => {
  it('App rendered!"', () => {
    const component = renderer.create(<App />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
