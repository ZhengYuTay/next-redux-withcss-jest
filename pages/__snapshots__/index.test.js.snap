// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App rendered!" 1`] = `
<div
  className="example"
>
  O Hai world!
</div>
`;
